### DISCUSSION TITLE
Error on GCP

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5289] [Q&A]
HI, I am constantly getting below error on the LOG

"MountVolume.SetUp failed for volume "my-bridge-bridge-token-*******" : secret "my-bridge-bridge-token-*****" not found"
"MountVolume.SetUp failed for volume "strimzi-cluster-operator-token-****" : secret "strimzi-cluster-operator-token-****" not found"

what is causing this error?  and how can I fix it ?

**Comment by scholzj** [comment]
I don't know ... that looks like service account secrets. Maybe your cluster does something special with the Service accounts and their token secrets? It is also not clear in which log did you found it.
